docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -267,10 +267,20 @@ Unless you originally installed Strapi with PostgreSQL, you need to install the
 
 `Path: ./my-project/`
 
+
+With npm:
+
 ```bash
 npm install pg --save
 ```
 
+With yarn:
+
+```bash
+yarn add pg
+```
+
+
```

Comment:
````suggestion

:::: tabs
::: tab npm

```bash
npm install pg --save
```
:::

::: tab yarn
```bash
yarn add pg
```
:::
::::

````

Could you please apply this suggestion? Then it will be ready for merge. Thank you! 🙂